kythe/release/cloudbuild/website-updater/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/kythe-repo/bazelisk-builder-client
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y \
```

Comment:
These should be a single docker `RUN` directive: https://docs.docker.com/develop/develop-images/instructions/#run

Reply:
Done.